app/controllers/MyProjects/DashboardController.php

```diff
@@ -127,10 +127,10 @@ function dashboard() {
 			} elseif ($this->request->session->getVar('entity_type')) {
 				$vs_entity_type = $this->request->session->getVar('entity_type');
 			}else{
-				$vs_entity_type = 'specimen';
+				$vs_entity_type = 's';
 			}
-			if (!in_array($vs_entity_type, ['specimen', 'media'])) {
-				$vs_entity_format = 'specimen';
+			if (!in_array($vs_entity_type, ['s', 'm'])) {
+				$vs_entity_type = 's';
```

Comment:
before vs_entity_format was set, now it is vs_entity_type (different variable names) Just want to make sure this is expected.

Reply:
Just to respond to this before merging, this was an error that is now fixed.